vision/detect/detect_test.go

```diff
@@ -57,30 +55,34 @@ func TestDetect(t *testing.T) {
 		if tt.local == nil {
 			continue
 		}
-
-		var buf bytes.Buffer
-		err := tt.local(&buf, "../testdata/"+tt.path)
-		if err != nil {
-			t.Fatalf("Local %s(%q): got %v, want nil err", tt.name, tt.path, err)
-		}
-		if got := buf.String(); !strings.Contains(strings.ToLower(got), strings.ToLower(tt.wantContain)) {
-			t.Errorf("Local %s(%q): got %q, want to contain %q", tt.name, tt.path, got, tt.wantContain)
-		}
+		t.Run(tt.name+"/local", func(t *testing.T) {
+			t.Parallel()
+			var buf bytes.Buffer
+			err := tt.local(&buf, "../testdata/"+tt.path)
+			if err != nil {
+				t.Fatalf("Local %s(%q): got %v, want nil err", tt.name, tt.path, err)
+			}
+			if got := buf.String(); !strings.Contains(strings.ToLower(got), strings.ToLower(tt.wantContain)) {
+				t.Errorf("Local %s(%q): got %q, want to contain %q", tt.name, tt.path, got, tt.wantContain)
+			}
+		})
 	}
 
 	for _, tt := range tests {
 		if tt.gcs == nil {
 			continue
 		}
-
-		var buf bytes.Buffer
-		err := tt.gcs(&buf, "gs://python-docs-samples-tests/vision/"+tt.path)
-		if err != nil {
-			t.Fatalf("GCS %s(%q): got %v, want nil err", tt.name, tt.path, err)
-		}
-		if got := buf.String(); !strings.Contains(strings.ToLower(got), strings.ToLower(tt.wantContain)) {
-			t.Errorf("GCS %s(%q): got %q, want to contain %q", tt.name, tt.path, got, tt.wantContain)
-		}
+		t.Run(tt.name+"/gcs", func(t *testing.T) {
+			t.Parallel()
+			var buf bytes.Buffer
+			err := tt.gcs(&buf, "gs://python-docs-samples-tests/vision/"+tt.path)
```

Comment:
nit: `if err := [...]; err != nil { ... }`

(here and elsewhere)

Reply:
Done.